site/content/docs/0.0/examples/grid/index.html

```diff
@@ -0,0 +1,206 @@
+---
+layout: examples
+title: Grid Template
+extra_css:
+- "grid.css"
+body_class: "py-4"
+aliases:
+- "/examples/grid"
+- "/docs/examples/grid"
+---
+
+<main>
+  <div class="container-fluid max-width-specific-tools">
```

Comment:
I'm not sure why we use `.max-width-specific-tools` and not `.max-width-public-website`